src/libs/actions/TransactionEdit.js

```diff
@@ -0,0 +1,39 @@
+import Onyx from 'react-native-onyx';
+import _ from 'underscore';
+import ONYXKEYS from '../../ONYXKEYS';
+
+/**
+ * Makes a backup copy of a transaction object that can be restored when the user cancels editing a transaction.
+ *
+ * @param {Object} transaction
+ */
+function createBackupTransaction(transaction) {
+    const newTransaction = {
+        ..._.omit(transaction, ['errorFields', 'isLoading']),
```

Comment:
That's a good point. I did this because I had some messy data locally and it kept getting errors stuck on the original transaction without any way to clear them out, so that's why I did this. I'll update this to stop omitting anything.

Reply:
@tgolen please remember to update this

Reply:
Please remember to do this